sqlserver/README.md

```diff
@@ -1,32 +1,59 @@
-# Sqlserver Integration
+# Microsoft SQL Server Check
 
-## Overview
+# Overview
```

Comment:
nitpick: this title (and all the other titles except for the name of the integration at the top) should probably be one level lower (i.e. `##`  instead of `#`)